docs/staking/testnet.rst

```diff
@@ -1,109 +1,33 @@
 Testnet Setup
 =============
 
+**Note:** THIS DOCUMENT IS OUT OF DATE
```

Comment:
Are you referring to this README or the installation link? Is this README still out of date?

Reply:
Yes - I've just semi-updated it for some commands which could be contextually useful, however we need to totally re-think what a local testnet is, like maybe we should be using containers, it's a hassle (more hassle than we want in a tutorial such as this) to deal with all the port updates if you want to genuinely run two tendermint instances locally